tests/test_v1.py

```diff
@@ -0,0 +1,466 @@
+    assert res['success'] is True
+    assert 'peers' in res
+    assert len(res['peers']) > 1
+
+
+def test_peer_peers_with_parameters():
+    parameters = {'status': 'OK'}
+    res = ARK.peers().peers(parameters)
+    assert res['success'] is True
+    assert 'peers' in res
+    statuses_are_ok = [peer['status'] == 'OK' for peer in res['peers']]
+    assert all(statuses_are_ok)
+
+
+def test_peer_version():
+    res = ARK.peers().version()
+    assert res['success'] is True
+    assert res['version'] == VERSION
+
+
+# Signature
+
+def test_signature_fee():
+    res = ARK.signatures().fee()
+    assert res['success'] is True
+    assert 'fee' in res
+
+
+@pytest.mark.skip('Need to set up an account and add transaction testing.')
+def test_signature_create():
+    assert False
+
+
+# Transaction
+
+def test_transaction_transaction():
+    res = ARK.transactions().transaction(env.TRANSACTION_ID)
+    assert res['success'] is True
+    assert 'transaction' in res
+
+
+def test_transaction_transactions():
+    res = ARK.transactions().transactions()
+    assert res['success'] is True
+    assert 'transactions' in res
+    assert len(res['transactions']) > 1
+
+
+def test_transaction_transactions_with_parameters():
+    parameters = {'limit': 1, 'offset': 0}
+    res = ARK.transactions().transactions(parameters)
+    assert res['success'] is True
+    assert 'transactions' in res
+    assert len(res['transactions']) == 1
+
+
+@pytest.mark.skip('Need to create transaction to test unconrfirmed transaction')
+def test_transaction_unconfirmed_transaction():
+    parameters = {'orderBy': 'timestamp:desc',
+                  'limit': 1}
+    res = ARK.transactions().transactions(parameters)
+    assert res['success'] is True
+    id_ = res['transactions'][0]['id']
+    res = ARK.transactions().unconfirmed_transaction(id_)
+    assert res['success'] is True
+    assert 'transaction' in res
+
+
+def test_transaction_unconfirmed_transactions():
+    res = ARK.transactions().unconfirmed_transactions()
+    assert res['success'] is True
+    assert 'transactions' in res
+
+
+@pytest.mark.skip('Need to set up an account and add transaction testing.')
+def test_transaction_create():
+    assert False
+
+
+# Transport
+
+def test_transport_list():
+    res = ARK.transport().list()
+    assert res['success'] is True
+    assert 'peers' in res
+
+
+@pytest.mark.skip('Server returns Expected type string but found type integer: #/ids')
+def test_transport_blocks_common():
+    ids = [env.BLOCK_ID]
+    res = ARK.transport().blocks_common(ids)
+    assert res['success'] is True
+    assert 'common' in res
+    assert 'lastBlockHeight' in res
+
+
+def test_transport_blocks_common_multiple():
+    ids = [env.BLOCK_ID, env.NEXT_BLOCK_ID]
+    res = ARK.transport().blocks_common(ids)
+    assert res['success'] is True
+    assert 'common' in res
+    assert 'lastBlockHeight' in res
+
+
+@pytest.mark.skip('Server not returning "success" obj in JSON.')
```

Comment:
how come it's not returning success in the json object?

Reply:
I don't know. I can test other peers and also see what happens on mainnet.